build-tools/packages/build-cli/src/lib/branches.ts

```diff
@@ -85,7 +85,7 @@ export function generateBumpVersionBranchName(
  */
 export function generateBumpDepsBranchName(
     bumpedDep: ReleaseGroup,
-    bumpType: VersionBumpTypeExtended,
+    bumpType: VersionBumpTypeExtended | "releasedDeps",
```

Comment:
Nit: Might be better to introduce a type alias with this definition so we can document the semantic distinction between this and standard `VersionBumpTypeExtended`.

Reply:
Agreed. Leaving type cleanup for a future PR.